frontend/src/app/sessions/session/tiling-window-manager/tiling-window-manager.component.html

```diff
@@ -0,0 +1,75 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="flex gap-0.5">
+  <ng-container *ngFor="let session of sessions">
+    <div
+      class="flex flex-col"
+      [ngClass]="{
+        height: (fullscreenService.isFullscreen$ | async) === false,
+        'h-screen': fullscreenService.isFullscreen$ | async
+      }"
+      [style.width]="session.width + 'px'"
+    >
+      <div
+        class="flex items-center justify-between gap-2 rounded-t bg-slate-100 p-1"
+      >
+        <span>
+          {{ session.version?.tool?.name }} {{ session.version?.name }},
+          {{ session.type }}
+          <span *ngIf="session.type === 'readonly'"
+            >(project {{ session.project!.name }})</span
+          >
+          {{ session.id }}
+        </span>
+
+        <div class="flex items-center">
+          <mat-icon
+            *ngIf="session.index !== 0"
+            class="cursor-pointer text-lg text-gray-600 hover:text-gray-800"
+            (click)="onLeftClick(session)"
```

Comment:
https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/566e3ce4-a722-4e7d-9337-50b99ccb68b2

In the video you can see that the session "jumps". This is the case because the session keep their sizes. I'm not sure if we should keep the original layout and just switch the sessions in the existing "boxes".